test/functional/admin/attachments_controller_test.rb

```diff
@@ -1,6 +1,8 @@
 require 'test_helper'
 
 class Admin::AttachmentsControllerTest < ActionController::TestCase
+  include LockedDocumentConcern
```

Comment:
Is this used to have the exception in scope? If so you can avoid the need for this by referencing the exception as `LockedDocumentConcern::LockedDocumentError` as including the module seems to apply we're using methods from the module.